tensorbay/cli/log.py

```diff
@@ -189,36 +239,75 @@ def _sort(self) -> None:
         self._sorted_commits.insert(index, commits)
         self._keys.insert(index, date)
 
-    def _generate_commits(self) -> Iterator[Commit]:
-        """Get the latest commit in commit list.
+    def _generate_commits_and_drafts(self) -> Iterator[Union[Commit, Draft]]:
+        """Get the latest commit or draft.
 
         Yields:
-            The latest commit.
+            The latest commit or draft.
 
         """
         while True:
             try:
-                latest_commit = self._sorted_commits[-1].pop(0)
+                latest_commit = self._sorted_commits[-1][0]
             except IndexError:
+                if self._sorted_drafts:
+                    yield from self._sorted_drafts[::-1]
                 return
-            if self._merge(latest_commit):
-                continue
-            yield latest_commit
-            self._sort()
+            latest_draft: Optional[Draft] = self._sorted_drafts[-1] if self._sorted_drafts else None
+            if latest_draft and latest_draft.updated_at > latest_commit.committer.date:
+                yield latest_draft
+                del self._sorted_drafts[-1]
+            else:
+                del self._sorted_commits[-1][0]
+                if self._merge(latest_commit):
+                    continue
+                yield latest_commit
+                self._sort()
 
-    def generate_commits(self) -> Iterator[str]:
-        """Get the the log of all commits.
+    def generate_commits_and_drafts(self) -> Iterator[str]:
+        """Get the the log of all commits and open drafts.
 
         Yields:
-            The commit log.
+            The commit or draft log.
 
         """
-        for commit in self._generate_commits():
-            commit_id = commit.commit_id
-            yield self._printer(commit, self._commit_id_to_branches.get(commit_id))
+        for element in self._generate_commits_and_drafts():
+            if isinstance(element, Draft):
```

Comment:
Is there a way not to use `isinstance`？
Would return whether the element is a Draft or not in `self._generate_commits_and_drafts()` save some time?